src/webgl/light.js

```diff
@@ -150,16 +150,17 @@ p5.prototype.directionalLight = function(v1, v2, v3, a, x, y, z) {
 
   var _x, _y, _z;
 
-  if(typeof arguments[arguments.length-1] === 'number'){
-    _x = arguments[arguments.length-3];
-    _y = arguments[arguments.length-2];
-    _z = arguments[arguments.length-1];
+  var argus = Array.prototype.slice.call(arguments);
```

Comment:
check out primitives.js for naming convention regarding arguments to Array.  variable should be var args instead of argus